src/main/java/io/jenkins/plugins/restlistparam/RestListParameterDefinition.java

```diff
@@ -171,7 +185,10 @@ public ParameterDefinition copyWithDefaultValue(final ParameterValue defaultValu
 
   @Override
   public ParameterValue createValue(final String value) {
-    RestListParameterValue parameterValue = new RestListParameterValue(getName(), value, getDescription());
+    RestListParameterValue parameterValue = values.stream().filter(item -> item.getDisplayValue().equals(value)).findFirst()
+      .map(item -> new RestListParameterValue(getName(), item.getValue(), item.getDisplayValue(), getDescription()))
+      .orElse(null);
```

Comment:
As described by some comments below, I am no fan of this in particular, as it seems kind of obsolete to send **and store** the '*displayValue*' within the RestListParameterValue (RLPV).
However, I am curious what you think of my proposal to expect the values index as the parameter of this function and therefor simplifying the access of the value to send/persist significantly.

(don't get me wrong, I like the streams API as you can see in the other classes of this plugin, I just dont understand why u need it here)